A01/COMP348-A1-AlexJeanFonz/data/Employees.txt

```diff
@@ -0,0 +1,15 @@
+0282828,Alex,Fourneaux,57000
+3430343,Jean,Robs,82000
+2672001,Alfonso,Fiorani,16000
```

Comment:
Because we're dealing with money, i think we should include the .00 precision in the salary